src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
+        super(color);
+        this.side=side;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
+        System.out.println("side: " + side + " units");
+    }
+
+    @Override
+    public double getArea() {
+        return side*side;
```

Comment:
There should be spaces around the multiplication operator for better readability: `return side * side;`.